src/frontends/pytorch/src/op/adaptive_max_pool2d.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_adaptive_max_pool2d(NodeContext& context) {
+    auto x = context.get_input(0);
```

Comment:
please check a number of inputs. The same comment for other operations.

Reply:
Same